### PR TITLE
Removed one level of nesting in page creation test

### PR DESCRIPTION
**Pull request by Marketionist**
**This pull request updates page creation integration test to remove one level of nesting**

### CODE REVIEW

**Comment by FinalAngel** [comment]
LGTM

**Comment by FinalAngel** [comment]
@Marketionist any idea why tests are failing here?

**Comment by Marketionist** [comment]
@FinalAngel I assume that it's because ```editPageLink``` doesn't get clicked properly. I'll push some fixes to see if it'll help

File: aldryn_events/tests/frontend/integration/specs/spec.events.crud.js [hunk 1]
```diff
@@ -81,30 +81,32 @@ describe('Aldryn Events tests: ', function () {
...
-                            }, eventsPage.mainElementsWaitTime);
-
-                            // validate test link text
-                            eventsPage.testLink.getText()
-                                .then(function (title) {
-                                expect(title).toEqual('Test');
-                            });
-                        }
-                    });
+                    eventsPage.slugErrorNotification.isPresent();
```
Comment (vxsx) [inline aldryn_events/tests/frontend/integration/specs/spec.events.crud.js]: you need to `return` the promise

Comment (vxsx) [inline aldryn_events/tests/frontend/integration/specs/spec.events.crud.js]: otherwise you are returning undefined which is resolved right away

**Pull request merged**